services/analysis-cached.js

```diff
@@ -85,6 +85,14 @@ const SQL_QUERIES = {
 
 const ALLOWED_PARAMS = {
   annual: ['adm0', 'adm1', 'adm2', 'threshold', 'forestType', 'landCategory'],
+  treeCoverDensity: [
+    'adm0',
+    'adm1',
+    'adm2',
+    'threshold',
+    'forestType',
+    'landCategory',
+  ],
```

Comment:
honest question, `treeCoverDensity` is a new param in the query...? or what is the purpose of adding this array here? trying to understand a bit more this logic.

Reply:
Anytime we need a different value in the dataset argument, we need to create a different object on `AllowedParams` object. As I needed to validate treeCoverDensity dataset, I've created this object.